src/odemis/driver/xt_client.py

```diff
@@ -0,0 +1,344 @@
+        size: (tuple of int)
+            (width, height) of the size in pixel.
+        """
+        self.client.set_selected_area(start_position, size)
+
+    def get_selected_area(self):
+        """
+        Returns the current selected area in x, y, width, height in pixels.
+        If selected area is not active it returns the stored selected area.
+        """
+        x, y, width, height = self.client.get_selected_area()
+        return x, y, width, height
+
+    def selected_area_info(self):
+        """Returns the unit and range of set selected area."""
+        return self.client.selected_area_info()
+
+    def reset_selected_area(self):
+        """Reset the selected area to select the entire image."""
+        self.client.reset_selected_area()
+
+    def set_scanning_size(self, size):
+        """
+        Set the size of the to be scanned area (aka field of view or the size, which can be scanned with the current
+        settings).
+
+        Parameters
+        ----------
+        size: (float)
+            size for X in [m]. Y is always X * 442/512
+        """
+        self.client.set_scanning_size(size)
+
+    def get_scanning_size(self):
+        """
+        Returns
+        -------
+        scanning_size: dict
+            Dictionary containing the 'x' and 'y' scanning size in [m].
+        """
+        x, y = self.client.get_scanning_size()
+        return x, y
+
+    def scanning_size_info(self):
+        """Returns the scanning size unit and range."""
+        return self.client.scanning_size_info()
+
+    def set_ebeam_spotsize(self, spotsize):
+        """
+        Setting the spot size of the ebeam.
+        Parameters
+        ----------
+        spotsize: float
+            desired spotsize, unitless [-]
+        """
+        self.client.set_ebeam_spotsize(spotsize)
+
+    def get_ebeam_spotsize(self):
+        """Get the current spotsize of the electron beam."""
+        return self.client.get_ebeam_spotsize()
+
+    def spotsize_info(self):
+        """Returns the unit and range of the spotsize. Unit is None means the spotsize is unitless."""
+        return self.client.spotsize_info()
+
+    def set_dwell_time(self, dwell_time):
+        """
+
+        Parameters
+        ----------
+        dwell_time: float
+            dwell time in seconds
+        """
+        return self.client.set_dwell_time(dwell_time)
+
+    def get_dwell_time(self):
+        """Returns the dwell time in seconds."""
+        return self.client.get_dwell_time()
+
+    def dwell_time_info(self):
+        """Returns the unit and range of the dwell time."""
+        return self.client.dwell_time_info()
+
+    def set_ht_voltage(self, voltage):
+        """
+        Set the high voltage.
+
+        Parameters
+        ----------
+        voltage: float
+            Desired high voltage value in volt [V].
+
+        """
+        self.client.set_ht_voltage(voltage)
+
+    def get_ht_voltage(self):
+        """Get the HT Voltage in [V]."""
```

Comment:
You always wrote down the unit and `Returns...`. I would stick with one version.